db/migrate/20200406183029_add_cols_to_pruebas.rb

```diff
@@ -0,0 +1,7 @@
+class AddColsToPruebas < ActiveRecord::Migration[6.0]
+  def change
+    add_column :pruebas, :nombre, :string
```

Comment:
I want to believe that a `prueba` will not be valid without a name. I you think that's the case, please add the db rules. Something like `null: false`